packages/office-ui-fabric-react/src/components/Image/ImagePage.tsx

```diff
@@ -66,12 +66,22 @@ export class ImagePage extends React.Component<IComponentDemoPageProps, {}> {
         dos={
           <div>
             <ul>
-              <li>Provide descriptive values for the alt text attribute for all images.</li>
+              <li>
+                Provide descriptive values for the alt text attribute for all images. If alt text isn't provided for images, the image information is inaccessible, for example, to people who cannot see and use a screen reader that reads aloud the information on a page, including the alt text for the visual image. Make the description brief, but accurate.
```

Comment:
try to keep lines under 140 characters long. makes it easier to comment

"including the alt text for the visual image" seems redundant. Reread this section. If possible to make it shorter and more concise.